docs/mkcloud.md

```diff
@@ -166,13 +183,16 @@ you can run the "prepare" step in mkcloud using the `cache_clouddata=1`
 environment variable set. This will create a cache under `$cache_dir` (set to
 `/var/cache/mkcloud` by default) of images and repositories needed during
 deployment. Make sure your `$cache_dir` partition has enough free space to
-store all the repos. You should always monitor the space on that partition and
-adjust accordingly.
+store all the repos. At this time, the cache will require around 23G. You
+should always monitor the space on that partition and adjust accordingly.
 
 Note that you will need to run the "prepare" step again (with VPN access) if
 your `$cloudsource` environment variable had changed. For example, if you are
-developing on "develcloud8" and "develcloud7" in parallel, you will need to run the
-"prepare" step for each separately.
+developing on "develcloud8" and "develcloud7" in parallel, you will need to run
+the "prepare" step for each separately.
+
+US-based employees will experience far better performance when building the
+cache initially if they use the Micro Focus VPN rather than the SUSE R&D one.
```

Comment:
I really would want to understand why this is the case? are you saying that you're using the SUSE R&D endpoint in NUE? or the one in provo? if you use the one in provo it should really not matter..